### PR TITLE
Pair prog contact form

### PR DESCRIPTION
**Pull request by Somdotta07**
-Create Contact form with coding partner using form tag
- Add styling with CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,5 +180,37 @@ <h2 class="about-me-heading">About Me</h2>
...
+               
+               <input type="email" id="mail" name="user_email" placeholder="Email addresss" required>
+           </li>
+           <li>
+               
+              <textarea name="user_msg" id="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+           </li>
+        </ul>
+        <button type="submit" >Get in touch</button>
+    </form>
```
Comment (george-shammar) [inline index.html]: - [ ] Name input should have maxlength="30" attribute. Kindly update the current validation.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @Somdotta07  

Good job so far!

:heavy_check_mark: You've done a pretty good job in implementing almost all the requirements. :clap: 

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Somdotta07) [reply index.html]: I just made it correct.. Thanks for your review.

**Review by AshakaE — APPROVED** [review]
### Hi @Somdotta07  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Somdotta07** [comment]
Thanks for the approval 😃

**Pull request merged**